comp/forwarder/defaultforwarder/transaction/transaction.go

```diff
@@ -201,7 +201,7 @@ type TransactionsSerializer interface {
 
 // Transaction represents the task to process for a Worker.
 type Transaction interface {
-	Process(ctx context.Context, client *http.Client) error
+	Process(ctx context.Context, config config.Component, client *http.Client) error
```

Comment:
I wonder if it wouldn't make more sense to make `config.Component` a field in the different `Transaction`-implementing structs? Just, to avoid pushing it down the call stack as a parameter. This might be something for later.